php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesPerformer.java

```diff
@@ -570,7 +614,7 @@ public void visit(DeclareStatement node) {
             if (CancelSupport.getDefault().isCancelled()) {
                 return;
             }
-            declareStatements.add(node);     
```

Comment:
As @junichi11 wrote, please, avoid changes in unrelated files and lines. The reason is that the Git history is then more difficult to read/follow, thanks. I think that the IDE can be configured to format/change only the changed lines (see IDE Options dialog).